### ISSUE TITLE
dnu commands uninstall error: The directory is not empty.

### ISSUE THREAD

**Issue by tugberkugurlu** [aspnet/dnx#1687]
I have installed a global package called `k-http`. Then, I got the below error when I ran `dnu commands uninstall k-http`:
```
> D:\downloads\HTML StarterKit>dnu commands uninstall k-http
> System.IO.IOException: The directory is not empty.
> 
>    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
>    at System.IO.Directory.DeleteHelper(String fullPath, String userPath, Boolean
>  recursive, Boolean throwOnTopLevelDirectoryNotFound)
>    at System.IO.Directory.Delete(String fullPath, String userPath, Boolean recur
> sive, Boolean checkHost)
>    at NuGet.PhysicalFileSystem.DeleteDirectory(String path, Boolean recursive)
>    at NuGet.PackageRepository.RemovePackage(PackageInfo package)
>    at Microsoft.Framework.PackageManager.UninstallCommand.PurgeOrphanPackages()
>    at Microsoft.Framework.PackageManager.UninstallCommand.Execute(String command
> Name)
>    at Microsoft.Framework.PackageManager.Program.<>c__DisplayClass3_13.<Main>b__
> 26()
>    at Microsoft.Framework.Runtime.Common.CommandLine.CommandLineApplication.Exec
> ute(String[] args)
>    at Microsoft.Framework.PackageManager.Program.Main(String[] args)
> --- End of stack trace from previous location where exception was thrown ---
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
>    at Microsoft.Framework.Runtime.Common.EntryPointExecutor.Execute(Assembly ass
> embly, String[] args, IServiceProvider serviceProvider)
>    at dnx.host.Bootstrapper.RunAsync(List`1 args)
>    at dnx.host.RuntimeBootstrapper.ExecuteAsync(String[] args)
>    at dnx.host.RuntimeBootstrapper.Execute(String[] args)
```

**Comment by BrennanConroy** [comment]
This might be related to #1568 
If you uninstall it uses the current directory and relative paths from current directory, that PR fixes it.

**Comment by BrennanConroy** [comment]
@tugberkugurlu 
I haven't been able to repro this yet. Can you do this consistently? Do you have multiple commands installed globally? Could you run dnu -v commands uninstall k-http?

**Comment by BrennanConroy** [comment]
@tugberkugurlu 
This might be fixed with 7b8e7bc5e723f704e9e1950ba2ecfcd814b49b3b, since I could not reproduce your error could you verify, else I will close as fixed.
